frontend/test/ui/test_install.py

```diff
@@ -0,0 +1,37 @@
+import os
+
+import pytest
+
+from pages.desktop.home import Home
+from pages.desktop.detail import Detail
+
+
+@pytest.mark.nondestructive
+@pytest.mark.skipif(os.environ.get('SKIP_INSTALL_TEST') is not None,
```

Comment:
Yes so the preference ```xpinstall.signatures.required', False``` cannot be set in firefox release or beta, but only dev and nightly. Currently dev, is being shipped as beta it seems, so this skips the install tests on release but runs them on nightly.